tracer/src/Datadog.Trace/Debugger/ImmutableDebuggerSettings.cs

```diff
@@ -72,13 +73,10 @@ public ImmutableDebuggerSettings(bool enabled, ProbeMode probeMode, string apiKe
         public static ImmutableDebuggerSettings Create(DebuggerSettings debuggerSettings) =>
             Create(
                 debuggerSettings.Enabled,
-                debuggerSettings.ProbeMode,
                 debuggerSettings.ApiKey,
                 debuggerSettings.RuntimeId,
                 debuggerSettings.ServiceName,
```

Comment:
Possible mistake - looks like we removed `ServiceName` from `ImmutableDebuggerSettings`?

Reply:
Yes, you are right, leftovers, cleaned at 10ea17673b64235f948ebde1f87ba0bee0454791